manual/manual/cmds/comp.etex

```diff
@@ -100,6 +100,18 @@ phase can be run directly, as in:
 The produced file has the executable bit set, and it manages to launch
 the bytecode interpreter by itself.
 
+The compiler is able to emit some information on its internal stages.
+Upon successful completion of the type checking procedure, type information is
+used to create an typed abstract syntax tree (AST). It contains complete
+information about the location and the specific type of each term in the
+source file. In case of unsuccessful type checking, a partial typed tree
+is created.
+The compiler can output this trees as compiled ".cmt" files for the
+implementation of the compilation unit and ".cmti" for signatures. This
+operation is activated by passing the key "-bin-annot" to the compiler (see
+the description of "-bin-annot" below).
```

Comment:
I think that this paragraph (`The compiler can output ... by passing the flag "-bin-annot"`) should come first, then we can explain what are typed AST (aka the first paragraph) to the reader with the context that we are detailing a compiler option. Otherwise, the reader may wonder why we are suddenly talking about typechecking.